group-actors/src/main/java/org/sunbird/actors/CreateGroupActor.java

```diff
@@ -66,6 +69,12 @@ private void createGroup(Request actorMessage) throws BaseException {
     if (CollectionUtils.isNotEmpty(reqMemberList)) {
       memberList.addAll(reqMemberList);
     }
+    if(CollectionUtils.isNotEmpty(memberList)){
+      boolean isUseridRedisEnabled = Boolean.parseBoolean(PropertiesCache.getInstance().getConfigValue(JsonKey.ENABLE_USERID_REDIS_CACHE));
+      if(isUseridRedisEnabled) {
+        deleteUserCache(memberList);
```

Comment:
am not sure why delete is needed explicitly.

Reply:
TODO list 4th point said, 
4. wipe off userId cache and when a read happens, let it re-populate.
So if read happens, we need to repopulate, cache should be deleted when member is added right? Same logic is applied in update group also.

Reply:
PlatformCache.set does the delete itself. Why explicit delete is needed? 
`def set(key: String, data: String, ttl: Int = 0): Unit = {
		val jedis = getConnection
		try {
			jedis.del(key)
			jedis.set(key, data)
			if (ttl > 0) jedis.expire(key, ttl)`

Reply:
As discussed, retaining the deleteCache feature.

Reply:
this is deleting userId based cache.